src/process.rs

```diff
@@ -146,27 +148,31 @@ pub async fn process(
         }
     });
 
-    //TODO: Replace with pid
-    Ok("OK\n".into())
+    Ok(format!("{}\n", pid))
 }
 
 fn push_possible_output(
     (stdout, stderr): (Option<String>, Option<String>),
     poll_data: &Arc<Mutex<Vec<PollData>>>,
 ) {
+    println!("{:?}|{:?}", stdout, stderr);
```

Comment:
?

Reply:
Debug statement accidentally left in. Removed.